app/controllers/pages_controller.rb

```diff
@@ -32,11 +32,22 @@ def search
       end
       @results = results
     end
+    if search_params[:flags] == "true"
```

Comment:
I'm not sure if we need anything for the search here.  Do you think it would be better to index it on ES and then we can sort the results by it for admins?

Reply:
Ok I will create a ticket to index flags in ES